README.md

```diff
@@ -163,80 +167,82 @@ Then, when importing Paragon's core SCSS in your MFE the import needs to begin w
-        description="Label of a close button on Alert component"
-      />
-      ```
-     
-   - For places where the display string has to be a plain JavaScript string use ``formatMessage``, this would require access to ``intl`` object from ``react-intl``, e.g.
-      
-      - For class components use ``injectIntl`` HOC
-
-          ```javascript
-          import { injectIntl } from 'react-intl';
-          
-          class MyClassComponent extends React.Component {
-            render() {
-              const { altText, intl } = this.props;
-              const intlAltText = altText || intl.formatMessage({
-                id: 'pgn.MyComponent.altText',
-                defaultMessage: 'Close',
-                description: 'Close label for Toast component',
-              });
-              
-              return (
-                <IconButton
-                  alt={intlCloseLabel}
-                  onClick={() => {}}
-                  variant="primary"
-                />
-              )
-            }
-          }
-          
-          export default injectIntl(MyClassComponent);
-          ```
-
-      - For functional components use ``useIntl`` hook
-
-          ```javascript 
-          import { useIntl } from 'react-intl';
-    
-          const MyFunctionComponent = ({ altText }) => {
-            const intls = useIntl();
-            const intlAltText = altText || intl.formatMessage({
-              id: 'pgn.MyComponent.altText',
-              defaultMessage: 'Close',
-              description: 'Close label for Toast component',
-            });
-    
-            return (
-              <IconButton
-                alt={intlCloseLabel}
-                onClick={() => {}}
-                variant="primary"
-              />
-            )
-          
-          export default MyFunctionComponent;
-          ```
-      
+
+   - For places where you need to display a string, and it's okay if it is a React element use `FormattedMessage`, e.g. (see [Alert](src/Alert/index.jsx) component for a full example)
+
+     ```javascript
+     import { FormattedMessage } from 'react-intl';
+     <FormattedMessage
+       id="pgn.Alert.closeLabel"
+       defaultMessage="Dismiss"
+       description="Label of a close button on Alert component"
+     />;
+     ```
+
+   - For places where the display string has to be a plain JavaScript string use `formatMessage`, this would require access to `intl` object from `react-intl`, e.g.
+
+     - For class components use `injectIntl` HOC
+
+       ```javascript
+       import { injectIntl } from 'react-intl';
+
+       class MyClassComponent extends React.Component {
+         render() {
+           const { altText, intl } = this.props;
+           const intlAltText =
+             altText ||
+             intl.formatMessage({
+               id: 'pgn.MyComponent.altText',
+               defaultMessage: 'Close',
+               description: 'Close label for Toast component',
+             });
```

Comment:
This definitely looks wrong, `altText` shouldn't start on the next line (see line 223 in this file below, for some reason the style is different there)

Reply:
I think there is still a slight difference in indentation between these two designs